pkg/compat/client.go

```diff
@@ -225,7 +223,47 @@ func (c client) Create(ctx context.Context, in runtime.Object) error {
 	}
 
 	start := time.Now()
+
+	// First create
 	err = c.Client.Create(ctx, obj)
+	// If no error, verify thing we created exist in cache
+	if err == nil {
+		for true {
+			kind := obj.GetObjectKind()
+			u, err := runtime.DefaultUnstructuredConverter.ToUnstructured(in)
+			if err != nil {
+				golog.Print(fmt.Sprintf("unstructured client did not understand object with err: %T", err))
+			}
+			uNs, found, err := unstructured.NestedString(u, "metadata", "namespace")
+			if err != nil || !found {
+				golog.Printf("[kind=%v] meta.namespace not found", kind)
+				break
+			}
+			uName, found, err := unstructured.NestedString(u, "metadata", "name")
+			if err != nil || !found {
+				golog.Printf("[kind=%v] meta.name not found", kind)
+				break
+			}
+			golog.Print(fmt.Sprintf("\n\n\n\n\n\n<<<[CREATE OBJ]>>> kind=[%v] key=[%v/%v]\n\n\n\n\n\n", kind, uNs, uName))
```

Comment:
Ignore this logger and log statements, will be removed but helped to understand while debugging.